recipes/plugins_library/plugin_image.py

```diff
@@ -29,15 +32,14 @@ def on_submit(self, label: Dict, asset_id: str) -> None:
 
         project_id = self.project_id
 
-        if len(issues_array) > 0:
+        if issues_array and len(issues_array) > 0:
```

Comment:
`issues_array` can be  undefined ?

Reply:
I don't think so but pylance does, It triggered an error, I think it is linked to the new typing of check_rule_on_label return

Reply:
Oh, okay, wanted to understand why you changed it. 
You can also try to type the return of `check_rule_on_label` to be `List[Optional[str]]`, I think this would solve all the pylance problems

Reply:
I tried but it conflict with `[f"There are too many BBox ({counter}) - Only 1 BBox of Object A accepted"]` line 20. I will leave it as is

Reply:
There are no errors if the `check_rule_on_label` is left unchanged (if only the type of return is added). Did you test the plugin ? Because in fact `return array.append()` returns `None`, not the array, so that's why pylint suggested that `issues_array` can be undefined : https://trinket.io/python/f7ad7f9864

<img width="1039" alt="image" src="https://user-images.githubusercontent.com/99664787/232047719-6780f0e3-6e25-4ad3-8cb7-dd7bdbd281bc.png">

Reply:
indeed